reference/7.0/Microsoft.PowerShell.Core/About/about_Windows_PowerShell_Compatibility.md

```diff
@@ -0,0 +1,94 @@
+---
+keywords: powershell,cmdlet
+locale: en-us
+ms.date: 1/10/2020
+online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_windows_powershell_compatibility?view=powershell-7&WT.mc_id=ps-gethelp
+schema: 2.0.0
+title: about_Windows_PowerShell_Compatibility
+---
+# About Group Policy Settings
+
+## SHORT DESCRIPTION
+
+Describes the Windows PowerShell Compatibility funtionality for PowerShell
+
+## LONG DESCRIPTION
+
+Windows PowerShell Compatibility funtionality means that for modules in
+the `%windir%\system32\WindowsPowerShell\v1.0\Modules` folder,
+if the module manifest doesn’t indicate that module is compatible with
+PowerShell Core then that module will be loaded in a background
+Windows PowerShell 5.1 process and using implicit remoting reflected
+into current PowerShell session.
+
+Windows PowerShell Compatibility funtionality can be invoked in 3 ways:
+
+1. Explicitly using module import with `-UseWindowsPowerShell` parameter
+
+```powershell
+Import-Module -Name ScheduledTasks -UseWindowsPowerShell
+```
+
+1. Implicitly using module import by module name or path
+
+```powershell
+Import-Module -Name ServerManager
+```
+
+1. Implicitly by command discovery / module autoload
+
+```powershell
+Get-WindowsFeature # this will autoload ServerManager module using Windows PowerShell Compatibility
+```
+
+To disable implicit Windows PowerShell Compatibility import (cases 2 and 3 above)
+one can use `DisableImplicitWinCompat` setting in PowerShell configuration file.
+
+```powershell
+$ConfigPath = "$PSHOME\DisableWinCompat.powershell.config.json"
+'{"DisableImplicitWinCompat" : "False", "Microsoft.PowerShell:ExecutionPolicy": "RemoteSigned"}' | Out-File -Force $ConfigPath
+pwsh -settingsFile $ConfigPath # Implicit Windows PowerShell Compatibility funtionality is Disabled in this PowerShell
+```
+
+When a first module is imported using Windows PowerShell Compatibility funtionality
+a `WinPSCompatSession` remoting session is created. This remoting session
+can be used for operations that do not work corrctly on de/serialized objects
+so that entire pipeline is executed in Windows PowerShell and only final result
+is returned.
+
+```powershell
+$s = Get-PSSession -Name WinPSCompatSession
+Invoke-Command -Session $s -ScriptBlock {"Running in Windows PowerShell version $($PSVersionTable.PSVersion)";Get-WinPSLiveObject | Set-WinPSLiveObject}
```

Comment:
is it worthwhile to use real cmdlets rather than ones that may not exist?
When I ran this snippet, I got the (somewhat expected) error that the command was not found.

Reply:
It would be nice to have a real pipeline example with simple setup, but I don't have one at the top of my head.
Placed cmdlets in example into a comment to avoid the error.